src/__tests__/gilded_rose_spec.js

```diff
@@ -72,9 +86,25 @@ describe("Gilded Rose", function() {
           shop.updateQuality();
           expect(shop.items[0].quality).toEqual(80);
         });
+
+        it("does not change if sellIn is less than 0", () => {
+          const items = [new Item("Sulfuras, Hand of Ragnaros", -1, 80)];
+          const shop = new Shop(items);
+          shop.updateQuality();
+          expect(shop.items[0].quality).toEqual(80);
+        });
       });
 
       describe("Backstage passes to a TAFKAL80ETC concert", () => {
+        it("increases by 1 if sellIn is equal or more than 11", () => {
```

Comment:
"What" increases by 1? Inconsistent way of writing test cases: some written sometimes with < or > signs. When using > 11 here why take 15 as test case?